frontend/hub/administration/collection-approvals/hooks/useApprovalActions.tsx

```diff
@@ -67,6 +67,21 @@ export function useApprovalActions(callback?: (collections: CollectionVersionSea
               ? t`Signature must be uploaded first`
               : undefined,
       },
+      {
+        type: PageActionType.Button,
+        selection: PageActionSelection.Single,
+        isPinned: true,
+        icon: ImportIcon,
+        label: t('View Import Logs'),
+        onClick: (collection) =>
+          pageNavigate(HubRoute.MyImports, {
```

Comment:
pageNavigate does not lead to any URL, because route for MyImports is defined in HubRoutes.tsx, but thats only constant, that we use for referencing the full route. The full route must be defined in useHubNavigation.tsx.

Reply:
Thanks for the review. I addressed this in https://github.com/ansible/ansible-ui/pull/1473, and tests are not going to work as well until #1473 is merged.